tests/bench/carma_fractalopticstest.txt

```diff
@@ -20,13 +20,13 @@
   8 0.166E+01 0.240E-02
   9 0.165E+01 0.100E-02
  10 0.163E+01 0.720E-03
-  1  1  0.250E+02  0.393E+00  0.922E+00  0.221E+01  0.549E+00  0.904E+00
-  2  1  0.248E+02  0.439E+00  0.910E+00  0.223E+01  0.536E+00  0.916E+00
-  3  1  0.215E+02  0.518E+00  0.863E+00  0.229E+01  0.520E+00  0.918E+00
-  4  1  0.148E+02  0.700E+00  0.775E+00  0.237E+01  0.499E+00  0.910E+00
-  5  1  0.919E+01  0.848E+00  0.757E+00  0.268E+01  0.566E+00  0.880E+00
-  6  1  0.457E+01  0.941E+00  0.763E+00  0.206E+01  0.716E+00  0.487E+00
-  7  1  0.257E+01  0.969E+00  0.766E+00  0.294E+01  0.900E+00  0.466E+00
-  8  1  0.130E+01  0.987E+00  0.765E+00  0.394E+01  0.988E+00  0.657E+00
-  9  1  0.847E+00  0.993E+00  0.761E+00  0.432E+01  0.996E+00  0.643E+00
- 10  1  0.141E+00  0.985E+00  0.713E+00  0.143E+01  0.996E+00  0.574E+00
+  1  1  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00
+  2  1  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00
+  3  1  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00
+  4  1  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00
+  5  1  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00
+  6  1  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00
+  7  1  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00
+  8  1  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00
+  9  1  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00
+ 10  1  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00  0.000E+00
```

Comment:
@cbardeen, These zeros in the fractalopticstest file seem suspicious.  We might need to take a closer look at this.